docker/install-docker-gcp-vm/docker-on-vm.yaml

```diff
@@ -0,0 +1,263 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+imports:
+  - http://cloudify.co/spec/cloudify/6.4.0/types.yaml
+  - plugin:cloudify-gcp-plugin?version= >=1.6.0
+  - plugin:cloudify-utilities-plugin?version= >=1.22.1
+  - plugin:cloudify-docker-plugin
+
+inputs:
+
+  region:
+    type: string
+    description: The GCP region to deploy the application in, such as europe-west1.
+    default: 'europe-west1'
+
+  zone:
+    type: string
+    description: The GCP zone to deploy the application in, such as europe-west1-b.
+    default: 'europe-west1-b'
+
+  image:
+    type: string
+    description: Centos image.
+    default: https://www.googleapis.com/compute/v1/projects/centos-cloud/global/images/centos-7-v20191210
+
+  instance_type:
+    type: string
+    description: A small GCP machine type.
+    default: n1-standard-2
+
+  agent_user:
+    description: >
+      The username of the agent running on the instance created from the image.
+    default: 'centos'
+
+  agent_key_name:
+    type: string
+    default: agent_key
+
+  env_name:
+    type: string
+    description: Control parameters for names in resources.
+    default: 'example'
+
+  docker_host:
+    type: string
+    default: { get_attribute: [vm, ip] }
```

Comment:
I don't think we need this input or the dsl_definition below. We should just put it directly in the node properties.